e2e/tests/accessSegregation_test.js

```diff
@@ -11,9 +11,9 @@ Scenario('Different user in the same local authority can see case created', asyn
   I.see(caseId);
 });
 
-Scenario('Different user in a different local authority cannot see case created', async ({I}) => {
-  await I.navigateToCaseDetailsAs(config.hillingdonLocalAuthorityUserTwo, caseId);
-  I.see('No cases found.');
+Scenario('Different user in a different local authority cannot see case created', async ({I, caseListPage}) => {
+  await I.signIn(config.hillingdonLocalAuthorityUserTwo);
+  caseListPage.verifyCaseIsNotAccessible(caseId);
```

Comment:
this is an async function, so could do with an `await`
```suggestion
  await caseListPage.verifyCaseIsNotAccessible(caseId);
```
and same in the other places you've used this refactor